manta-trusted-setup/src/util.rs

```diff
@@ -52,7 +51,7 @@ pub trait HasDistribution {
 /// [`Deserializer`] `trait`.
 ///
 /// [`CanonicalDeserialize`]: manta_crypto::arkworks::serialize::CanonicalDeserialize
-pub trait Serializer<T> {
+pub trait Serializer<T, M = ()> {
```

Comment:
Why the new parameter?

Reply:
This is part of making it a markable trait that lets you give custom implementations even when `T` is the same type.